lib/app/features/chat/views/pages/new_chat_modal/new_chat_modal.dart

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: ice License 1.0
+
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+import 'package:ion/app/components/button/button.dart';
+import 'package:ion/app/components/inputs/search_input/search_input.dart';
+import 'package:ion/app/components/screen_offset/screen_side_offset.dart';
+import 'package:ion/app/extensions/extensions.dart';
+import 'package:ion/app/features/chat/views/pages/new_chat_modal/components/new_chat_initial_view/new_chat_initial_view.dart';
+import 'package:ion/app/router/components/navigation_app_bar/navigation_app_bar.dart';
+import 'package:ion/app/router/components/navigation_app_bar/navigation_close_button.dart';
+import 'package:ion/app/router/components/sheet_content/sheet_content.dart';
+import 'package:ion/generated/assets.gen.dart';
+
+class NewChatModal extends StatelessWidget {
+  const NewChatModal({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return SheetContent(
+      topPadding: 0,
+      body: Column(
+        children: [
+          NavigationAppBar.modal(
+            showBackButton: false,
+            title: Text(context.i18n.new_chat_modal_title),
+            actions: [NavigationCloseButton(onPressed: () => context.pop())],
+          ),
+          SizedBox(height: 9.0.s),
+          ScreenSideOffset.small(
+            child: Column(
+              children: [
+                const Row(
+                  children: [
+                    Expanded(
+                      child: SearchInput(
+                        textInputAction: TextInputAction.search,
+                      ),
+                    ),
+                  ],
+                ),
+                SizedBox(height: 12.0.s),
+                Row(
+                  children: [
+                    Expanded(
```

Comment:
These Expanded widgets look similiar, only Icon, OnPressed and Label are different. Consider extracting to a separate widget.